transmart-copy/src/main/groovy/org/transmartproject/copy/Database.groovy

```diff
@@ -187,6 +188,39 @@ class Database implements AutoCloseable {
         log.info 'Done.'
     }
 
+    Set<Table> getChildTables(Table parentTable) {
+        log.debug "Getting child tables for: ${parentTable} ..."
+        def queryResult = jdbcTemplate.queryForList("""
+            SELECT cn.nspname as childSchema, c.relname AS childTable
+            FROM pg_inherits
+            JOIN pg_class AS c ON (inhrelid=c.oid)
+            JOIN pg_catalog.pg_namespace cn ON (c.relnamespace=cn.oid)
+            JOIN pg_class as p ON (inhparent=p.oid)
+            JOIN pg_catalog.pg_namespace pn ON (p.relnamespace=pn.oid)
+            WHERE pn.nspname || '.' || p.relname='${parentTable}';
+        """)
+        queryResult.collect { Map<String, Object> resultRow ->
```

Comment:
Why not the more Java-like style as in `getColumnMetadata`? The class is annotated with `@CompileStatic`, here we suddenly use the type checking.